.buildkite/pipeline.yml

```diff
@@ -1,77 +1,144 @@
 x-common-params:
-  &publish-android-artifacts-docker-container
-  docker#v3.8.0:
-    image: "public.ecr.aws/automattic/android-build-image:v1.1.0"
-    propagate-environment: true
-    environment:
-      # DO NOT MANUALLY SET THESE VALUES!
-      # They are passed from the Buildkite agent to the Docker container
-      - "AWS_ACCESS_KEY"
-      - "AWS_SECRET_KEY"
+  - &gb-mobile-docker-container
+    docker#v3.8.0:
+      image: "public.ecr.aws/automattic/gb-mobile-image:latest"
+      environment:
+        - "CI=true"
+        # Allow WP-CLI to be run as root, otherwise it throws an exception.
+        # Reference: https://git.io/J9q2S
+        - "WP_CLI_ALLOW_ROOT=true"
```

Comment:
I extracted this from a step. It's currently unused because I'm adding to the image locally. I hope all my local changes can eventually go upstream.